easybuild/easyconfigs/n/NCCL/NCCL-2.16.2-GCCcore-12.2.0-CUDA-12.0.0.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'Tarball'
+
+name = 'NCCL'
+version = '2.16.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/nccl'
+description = """The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
+communication primitives that are performance optimized for NVIDIA GPUs."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = [
+    'https://developer.download.nvidia.com/compute/redist/nccl/v%(version)s/'
+]
+sources = ['%(namelower)s_%(version)s-1+cuda%(cudashortver)s_%(arch)s.txz']
+checksums = [{
+    '%(namelower)s_%(version)s-1+cuda%(cudashortver)s_x86_64.txz':
+        'fb630e2d244cdc8cf3fe2cbe07cbf0a50a5213ba546edc717bdba5d5ac757b87',
+    '%(namelower)s_%(version)s-1+cuda%(cudashortver)s_ppc64le.txz':
+        'daac3b7de8d0091369cb59b59d011b494c2f16403c80478c15b97d88a9895651',
+    '%(namelower)s_%(version)s-1+cuda%(cudashortver)s_aarch64.txz':
+        'fb630e2d244cdc8cf3fe2cbe07cbf0a50a5213ba546edc717bdba5d5ac757b87',
```

Checksum is incorrect here (copy-paste from `x86_64`:

```suggestion
        '738d22d216266cd039610d0b87e008fcbba8ec9a93a626b958be96583d8b8b8a',

---

My bad, fixed.